xbmc/interfaces/python/PythonInvoker.cpp

```diff
@@ -224,6 +230,22 @@ bool CPythonInvoker::execute(const std::string &script, const std::vector<std::s
   if (m_argv != NULL)
     PySys_SetArgv(m_argc, m_argv);
 
+  if (item)
+  {
+    CFileItemPtr copiedItem = CFileItemPtr(new CFileItem(*item.get())); //use a copy of the item, so the python script cannot manipulate the item directly
+
+    XBMCAddon::xbmcgui::ListItem* arg = new XBMCAddon::xbmcgui::ListItem(copiedItem);
+    m_item = PythonBindings::makePythonInstance(arg,
+                                                &PythonBindings::TyXBMCAddon_xbmcgui_ListItem_Type.pythonType,
```

Comment:
Do you really need the explicit API type here? AFAIK `PythonBindings::makePythonInstance(arg, true)` should be capable to determine this on its own as long as `arg` is an instance of a class derived from `XBMCAddons::AddonClass`.